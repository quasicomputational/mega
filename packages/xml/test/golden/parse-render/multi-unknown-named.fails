<!DOCTYPE scene SYSTEM "example:entity-set">
<scene
   >
  &foobarbazunknownandundefined;.
  Furthermore, &spameggsandham;.
</scene>
