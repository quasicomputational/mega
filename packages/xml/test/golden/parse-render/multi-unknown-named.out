Error: Unknown entity 'foobarbazunknownandundefined' at (4:4-4:32).
Error: Unknown entity 'spameggsandham' at (5:17-5:31).
